src/components/App.js

```diff
@@ -9,7 +9,7 @@ const LocaleContext = React.createContext({})
 const App = ({ children, title, description, locale }) => {
   return (
     <LocaleContext.Provider value={{ locale }}>
-      <div className="main">
+      <div className="main" lang={locale}>
```

Comment:
🙌 

Would it make sense to set `lang` on the `body`, as a side note?

Reply:
That would probably be better. Will update in a future PR!

Reply:
Or `html`, tbh. `html` being the parent of all elements, it'll make `:lang` work everywhere. Notably, since `:lang` looks for the nearest parent with the `lang` attribute set, we could have mixed language content if e.g. only part of the page were translated. Not sure that's useful for us, just noticing some consequences of how the spec is set up.